project_generation/content/templates/api/main.go.tmpl

```diff
@@ -45,9 +65,7 @@ func run() error {
 	case err := <-svcErrors:
 		return errors.Wrap(err, "service error received")
```

Comment:
I don't know if we should try to do the graceful shutdown if there is an error too?

Reply:
leave this one for another PR (maybe)

Reply:
OK, hint added